test/Engine/ProtoTestFx/ProtoTestFx.csproj

```diff
@@ -9,28 +9,28 @@
     <RootNamespace>ProtoTestFx</RootNamespace>
     <AssemblyName>ProtoTestFx</AssemblyName>
   </PropertyGroup>
-  <PropertyGroup>
-    <SignAssembly>false</SignAssembly>
-  </PropertyGroup>
-  <PropertyGroup>
-    <AssemblyOriginatorKeyFile>ProtoTestFx.snk</AssemblyOriginatorKeyFile>
```

Comment:
I see no reason to believe these strong name keys are still used. We sign our binaries with an ADSK key.